core-web/libs/sdk/react/src/lib/components/DotEditableText/DotEditableText.tsx

```diff
@@ -0,0 +1,135 @@
+import { Editor } from '@tinymce/tinymce-react';
+import { useEffect, useRef, useState } from 'react';
+
+import {
+    isInsideEditor as isInsideEditorFn,
+    postMessageToEditor,
+    CUSTOMER_ACTIONS,
+    DotCmsClient
+} from '@dotcms/client';
+
+import { DotEditableTextProps, TINYMCE_CONFIG } from './utils';
+
+export function DotEditableText({
+    mode = 'plain',
+    format = 'text',
+    contentlet,
+    fieldName = ''
```

Comment:
We need to log an error and stop the rendering if title is not passed.

Reply:
Same with contentlet, we need to be more defensive.

Reply:
Probably even show something in the UI in dev mode saying that needs this and that.

Reply:
I'll do it!

Reply:
@fmontes  I'll conso.log an error.

We should not display anything in the UI in dev mode because the UVE should be a reflection of the real page since there is no "Preview" button.